site/docs/components/static-list/usage.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Using the component
+
+### When to use
+
+- To present a set of static list items. It can be used with a File drop zone to show the list of files added. It can also be used to create a shopping cart list.
```

Comment:
```suggestion
- To present a set of static list items. It can be used with a [`FileDropZone`](/salt/components/file-drop-zone) to show the list of files added. It can also be used to create a shopping cart list.
```
?